internal/store/elasticsearch/discovery_search_repository_test.go

```diff
@@ -219,6 +248,12 @@ func TestSearcherSearch(t *testing.T) {
 				for i, res := range test.Expected {
 					assert.Equal(t, res.Type, results[i].Type)
 					assert.Equal(t, res.AssetID, results[i].ID)
+					if test.Config.Flags.EnableHighlight {
+						actualHighlightMap := results[i].Data["_highlight"].(map[string][]string)
+						for key, value := range res.Data["_highlight"].(map[string][]string) {
+							assert.Equal(t, value, actualHighlightMap[key])
+						}
```

Comment:
```suggestion
						assert.Equal(t, res.Data["_highlight"], results[i].Data["_highlight"])
```
can you check if this works?

Reply:
done